internal/cli/login.go

```diff
@@ -85,28 +91,13 @@ func loginCmd(cli *cli) *cobra.Command {
 
 // RunLoginAsUser runs the login flow guiding the user through the process
 // by showing the login instructions, opening the browser.
-// Use `expired` to run the login from other commands setup:
-// this will only affect the messages.
-func RunLoginAsUser(ctx context.Context, cli *cli, expired bool) (Tenant, error) {
-	message := fmt.Sprintf(
-		"%s\n\n%s\n\n",
-		"✪ Welcome to the Auth0 CLI 🎊",
-		"If you don't have an account, please create one here: https://auth0.com/signup.",
-	)
-
-	if expired {
-		message = "Please sign in to re-authorize the CLI."
-		cli.renderer.Warnf(message)
-	} else {
-		cli.renderer.Output(message)
-	}
```

Comment:
Extracting the error messaging out of the `RunLoginAsUser` is an intentional decision to keep the function focused. Further, each invocation of `RunLoginAsUser` comes with a unique message, that is, there is _no_ duplication of messages with the removal of this code. This allows the messaging to stay specific to the situation and make the function more focused.